scripts/waitForSync.py

```diff
@@ -0,0 +1,63 @@
+import os
```

Comment:
Not sure, but was wondering if we could reuse the script from post-merge tests, what do you think? Probably not needed at this point, I guess?

Reply:
I reached the limit of optimizing to various ise cases SH script and python gave me more flexibility and went quickly with rewrite.

I have no issue reusing existing one and keep them identical to some extent!